processor/stream/package_tests/metadata_attrs_test.go

```diff
@@ -73,7 +73,7 @@ func metadataProcSetup() *tests.ProcessorSetup {
 			intakeTestProcessor{Processor: stream.Processor{MaxEventSize: lrSize}}},
 		SchemaPath: "../../../docs/spec/v2/metadata.json",
 		TemplatePaths: []string{
-			"../../../_meta/fields.common.yml",
+			"../../../model/transaction/_meta/fields.yml",
```

Comment:
Can you please add a NOTE here for why transaction fields are used and add a TODO comment for when we refactor the package tests.

Reply:
I added a comment, but wouldn't know what to say in a TODO. Feel free to add a GH comment-suggestion thingy.

Reply:
How about something along the lines of:
```
TODO: move metadata package tests into event specific tests when refactoring package tests
```

Reply:
```suggestion
			// TODO: move metadata package tests into event specific tests when refactoring package tests
			"../../../model/transaction/_meta/fields.yml",
```